packages/dapp/src/components/Navbar/SubMenuComponent.tsx

```diff
@@ -51,6 +58,28 @@ const SubMenuComponent = ({
     }
   }
 
+  const handleClick = (el: MenuListItem) => {
+    if (el.triggerSubMenu) {
+      onOpenNavbarSubMenu(el.triggerSubMenu);
+      trackEvent({
+        category: TrackingCategory.SubMenu,
+        action: TrackingAction.OpenMenu,
+        label: `open_submenu_${el.triggerSubMenu.toLowerCase()}`,
+        data: {
+          [TrackingEventParameter.Menu]: el.triggerSubMenu,
+          [TrackingEventParameter.PrevMenu]: prevMenu,
+        },
+        disableTrackingTool: [EventTrackingTool.Raleon, EventTrackingTool.ARCx],
+      });
+    } else {
```

Comment:
minor suggestion: `else` isn't required here 😄

Reply:
Actually, it is needed the way I have set it up. The idea is that you can pass a "triggerSubMenu" to such a MenuItem. This may be any "menuKey" as we called it, such as Wallet, Chains, Themes.. That way, another sub-menu can be opened. 

Other times when a MenuItem serves to trigger some function, else is actually needed in order to make that menu work. E.g. Opening a tab with some url